quickstart/README.md

```diff
@@ -7,17 +7,32 @@
 
 ## Lab
 
+The container config of [Lobsters](https://github.com/jcs/lobsters) is
+[here](lobsters/). It is a test configuration with a local sqlite db
+inside the single container.
+
 Run the Lobsters app container
 
-<!-- dont-START bash -->
+<!-- START bash -->
 ```
 kubectl run lobsters --image=gcr.io/google-samples/lobsters:latest
 ```
-<!-- dont-END bash -->
+<!-- END bash -->
 
-<!-- Test this in the meantime START bash
-kubectl run nginx --image=nginx
-  END bash -->
+```
+deployment "lobsters" created
+```
+
+Get deployments
+
+```
+kubectl get deployments
```

Comment:
Thought on style: it would be great to make it clear which sections where things someone types and which were results. Sometimes that's done by prepending commands with "$" to show that they're run in a shell... but that means no tripple-click for cut and paste.